### PR TITLE
[MRG] Cleaning of docs by removing overlaps.

### PR DESCRIPTION
**Pull request by jaeilepp**
Visualization examples so far.

### CODE REVIEW

File: mne/forward/_field_interpolation.py [hunk 1]
```diff
@@ -198,7 +198,7 @@ def _as_meg_type_evoked(evoked, ch_type='grad', mode='fast'):
 
     info_from = pick_info(evoked.info, pick_from, copy=True)
     info_to = pick_info(evoked.info, pick_to, copy=True)
-    mapping = _map_meg_channels(info_from, info_to, mode='fast')
+    mapping = _map_meg_channels(info_from, info_to, mode=mode)
```
Comment (jaeilepp) [inline mne/forward/_field_interpolation.py]: Noticed that ``mode`` wasn't used.

**Comment by jaeilepp** [comment]
Looking at the examples, I noticed that basically all input/output examples are covered somewhere in the tutorials. I'm just a bit hesitant to remove the whole section as it has the IO operations neatly under one headline and the examples are not computationally heavy. Maybe we can leave it as it is.
Wdyt?

**Comment by agramfort** [comment]
Creating MNE objects from data arrays is not covered in tuto as well.

keep the section but add refs to tutorials.

add some ref:`tut_xxx` in examples/io/README.txt to the tutorials.

**Comment by jaeilepp** [comment]
Ok, here's the first iteration round. I removed the introductory examples as they are now covered in tutorials. I also feel that maybe the ``datasets`` examples should be made into tutorials by adding a bit more narration.
Ready for review if it renders right on circle.

**Comment by agramfort** [comment]
LGTM

next in a next PR?

File: examples/io/plot_read_evoked.py [hunk 2]
```diff
@@ -29,3 +30,7 @@
 
 # Show result as a 2D image (x: time, y: channels, color: amplitude)
 evoked.plot_image(exclude=[])
+
+###############################################################################
+# Use :func:`mne.Evoked.save` or :func:`mne.write_evokeds` to write the evoked
+# responses to a file.
```
Comment (Eric89GXL) [inline examples/io/plot_read_evoked.py]: we should probably advertise `mne.write_evokeds`, too

File: tutorials/plot_visualize_epochs.py [hunk 3]
```diff
@@ -48,5 +48,7 @@
 epochs.plot_image(97)
 
 # You also have functions for plotting channelwise information arranged into a
-# shape of the channel array,
-epochs.plot_topo_image(vmin=-450, vmax=450)
+# shape of the channel array, The image plotting uses automatic scaling by
```
Comment (Eric89GXL) [inline tutorials/plot_visualize_epochs.py]: period instead of comma

File: tutorials/plot_visualize_evoked.py [hunk 4]
```diff
@@ -25,19 +25,20 @@
 # of the categories by passing the argument ``condition`` to
 # :func:`mne.read_evokeds`. To make things more simple for this tutorial, we
 # read each instance to a variable.
-left_auditory = evoked[0]
-right_auditory = evoked[1]
-left_visual = evoked[2]
-right_visual = evoked[3]
+evoked_laud = evoked[0]
```
Comment (Eric89GXL) [inline tutorials/plot_visualize_evoked.py]: evoked_l_aud?

**Comment by Eric89GXL** [comment]
Otherwise LGTM

File: examples/io/plot_read_evoked.py (hunk 2 above)
Comment (jaeilepp) [inline examples/io/plot_read_evoked.py]: It's advertised in the comment.

**Comment by jaeilepp** [comment]
Maybe this can be merged now. One of the tutorials was using a surprisingly lot of memory and made circle fail. This includes a fix to that.

**Comment by agramfort** [comment]
can you see why travis complains?

**Comment by Eric89GXL** [comment]
Now you need a rebase

File: tutorials/plot_epoching_and_averaging.py [hunk 5]
```diff
@@ -41,7 +41,15 @@
 # attention in aligning the events correctly with the raw data.
 events = mne.find_events(raw)
 print(events)
-mne.viz.plot_events(events, raw.info['sfreq'], raw.first_samp)
+
+# Plot the events to get an idea of the paradigm
+# Specify colors and an event_id dictionary for the legend.
+event_id = {'aud_l': 1, 'aud_r': 2, 'vis_l': 3, 'vis_r': 4, 'smiley': 5,
```
Comment (jona-sassenhagen) [inline tutorials/plot_epoching_and_averaging.py]: Use HEDs:
`{'aud/l': 1, 'aud/r': 2, 'vis/l': 3, 'vis/r' ...`

**Comment by jona-sassenhagen** [comment]
I don't understand some of these. Why, for example, is the decoding across time generalization across conditions example being removed? Why CORRMAP? I don't see replacements.

**Comment by Eric89GXL** [comment]
They should already be in tutorials from an earlier PR, can you check?

**Comment by jona-sassenhagen** [comment]
Here? http://martinos.org/mne/dev/tutorials.html
The links to decoding stuff are all dead.

**Comment by jaeilepp** [comment]
Check the circle builds. ICA tutorial in preprocessing should have the CORRMAP.

**Comment by agramfort** [comment]
tutorials/plot_sensors_decoding.py

File: tutorials/plot_epoching_and_averaging.py (hunk 5 above)
Comment (Eric89GXL) [inline tutorials/plot_epoching_and_averaging.py]: Let's tackle this is a subsequent PR since we do this in a lot of places currently

**Comment by Eric89GXL** [comment]
Thanks @jaeilepp I think you got everything, @jona-sassenhagen feel free to rebase your current HED PR to tackle that over there

**Pull request merged**